Jenkinsfile_nightly

```diff
@@ -11,7 +11,24 @@ def type = "angular"
 def product = "ccd"
 def app = "case-management-web"
 
+List<LinkedHashMap<String, Object>> secrets = [
+    secret('ccd-caseworker-autotest-email', 'CCD_CASEWORKER_AUTOTEST_EMAIL'),
+    secret('ccd-caseworker-autotest-password', 'CCD_CASEWORKER_AUTOTEST_PASSWORD'),
```

Comment:
Technically I don't think we need these in nightly since we're not running any tests on here that need it.  But we may in future so not a train smash.

Reply:
We need this for the nightly crossbrowser tests